mmpose/datasets/pipelines/pose3d_transform.py

```diff
@@ -115,6 +115,141 @@ def __call__(self, results):
         return results
 
 
+@PIPELINES.register_module()
+class ImageCoordinateNormalization:
+    """Normalize the 2D joint coordinate with image width and height. Range [0,
+    w] is mapped to [-1, 1], while preserving the aspect ratio.
+
+    Args:
+        item (str): The name of the pose to normalize.
+        camera_param (dict|None): The camera parameter dict. See the camera
+            class definition for more details. If None is given, the camera
+            parameter will be obtained during processing of each data sample
+            with the key "camera_param".
+    Required keys:
+        item
+    Modified keys:
+        item
+    """
+
+    def __init__(self, item, camera_param=None):
```

Comment:
It looks a little strange that the class name is "ImageCoordinateNormalization" but the camera_param is needed.
I suggest this transform get h and w from the results. And in the dataset's __getitem__, we put image size information into the fetched sample, from either camera parameters or loaded images (we will extend to RGB based 3D pose estimation in the future)